contracts/src/v0.8/llo-feeds/v0.5.0/test/mocks/MockFeeManager.sol

```diff
@@ -0,0 +1,351 @@
+// SPDX-License-Identifier: MIT
```

Comment:
How we generate the bindings to these mock contracts?
Maybe we should add to the documentation of /deployments/data-streams